app/src/androidTest/java/com/github/h3lp3rs/h3lp/professional/ProMainActivityTest.kt

```diff
@@ -72,4 +80,9 @@ class ProMainActivityTest : H3lpAppTest() {
         clickingOnButtonWorksAndSendsIntent(ForumCategoriesActivity::class.java, withId(R.id.faq_button))
     }
 
+    /*@Test
```

Comment:
If it doesn't work on Cirrusm it would be good to add a comment saying so (or explain why it's commented out)